lib/clarkson-core-deprecated.php

```diff
@@ -34,57 +34,14 @@ public function __construct() {
 		}
 	}
 
-	/**
-	 * Get theme objects.
-	 *
-	 * @return array
-	 */
-	public function get_theme_objects() {
-		$objects       = array();
-		$theme_objects = array();
-
-		// Load deprecated post-objects folder.
-		$theme_deprecated_objects_path = get_template_directory() . '/post-objects';
-		if ( is_dir( $theme_deprecated_objects_path ) ) {
-			_doing_it_wrong( __METHOD__, 'The ' . esc_html( $theme_deprecated_objects_path ) . " folder is deprecated. Please use 'wordpress-objects'.", '0.2.0' );
-			$theme_objects = array_merge( $this->get_objects_from_path( $theme_deprecated_objects_path ), $theme_objects );
-		}
-
-		/**
-		 * @hook clarkson_available_objects_paths
-		 * @deprecated This method of loading objects is no longer used, unless the clarkson_core_autoload_theme_pre_020 filter is enabled.
-		 * @since 0.1.0
-		 * @param {string[]} $theme_objects Objects available in Clarkson Core to be loaded.
-		 * @return {string[]} Files that should be included to make the object available to Clarkson_Core_Objects.
-		 */
-		$theme_objects = apply_filters( 'clarkson_available_objects_paths', $theme_objects );
-
-		// Load classes.
-		foreach ( $theme_objects as $object_name => $object_path ) {
-			if ( strpos( $object_name, '_tax_' ) !== false ) {
-				$object_name = strtolower( $object_name );
-			}
-
-			if ( in_array( $object_name, $objects, true ) ) {
-				continue;
-			}
-
-			include_once $object_path;
-			$objects[] = $object_name;
-		}
-		return $objects;
-	}
-
 	/**
 	 * Get map deprecated filters.
 	 *
 	 * @return array Deprecated filters.
 	 * @internal
 	 */
 	public function get_map_deprecated_filters() {
-		return array(
-			'clarkson_twig_functions' => 'yalla_twig_functions',
-		);
+		return array();
```

Comment:
Van mij mag dit hele bestand wel weg? Of is er een reden om dat niet te doen?

Reply:
Ik ging er vanuit dat dit nog een soort van bij het framework hoorde voor Clarkson. Mochten we weer deprecated filters o.i.d. krijgen, kunnen we dat hier kwijt. Maar w.m.b. kan dit weg.

Reply:
Ja, als we dit nog een keer nodig hebben kunnen we dit netter oplossen. Zie #94